javascript/ql/src/semmle/javascript/HTML.qll

```diff
@@ -159,6 +159,40 @@ module HTML {
      * if it can be determined.
      */
     Script resolveSource() { result.getFile().getAbsolutePath() = resolveSourcePath() }
+
+    /**
+     * Gets the inline script of this script element, if any.
+     */
+    private InlineScript getInlineScript() {
+      exists(string f, Location l1, int sl1, int el1, Location l2, int sl2, int el2 |
+        getLocation() = l1 and
+        result.getLocation() = l2 and
+        l1.hasLocationInfo(f, sl1, _, el1, _) and
+        l2.hasLocationInfo(f, sl2, _, el2, _)
+        |
+        (
+          sl1 = sl2 and l1.getStartColumn() < l2.getStartColumn()
```

Comment:
You can just pull start and end column out of the `hasLocationInfo` relation the same way as you did for `sl1` and `sl2`.